### DISCUSSION TITLE
My account is suspended

### DISCUSSION THREAD

**Discussion by jayzgame99** [community/community#129144] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My Account got suspended and still no response from support team.

Ticket ID: **2839418**
Email: **PII removed by Community Moderator**

Git Team, please response!

Thank you!

**Comment by will-bvs** [comment]
Me too.
No response was received.

https://github.com/orgs/community/discussions/129048

**Comment by LiteBrite82** [comment]
Hi @jayzgame99 and @piszz 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
